package/MDAnalysis/coordinates/RDKit.py

```diff
@@ -101,4 +135,113 @@ def __init__(self, filename, **kwargs):
+
+        Parameters
+        -----------
+        obj : AtomGroup or Universe
+        """
+        try:
+            from rdkit import Chem
+        except ImportError:
+            raise ImportError('RDKit is required for RDKitConverter but '
+                              'is not installed. Try installing it with \n'
+                              'conda install -c conda-forge rdkit')
+        try:
+            # make sure to use atoms (Issue 46)
+            ag = obj.atoms
+        except AttributeError as e:
+            raise TypeError("No `atoms` attribute in object of type {}, "
+                            "please use a valid AtomGroup or Universe".format(
+                            type(obj))) from e
+
+        mol = Chem.RWMol()
+        atom_mapper = {}
+
+        for atom in ag:
+            try:
+                element = atom.element
+            except NoDataError:
+                # guess atom element
+                # capitalize: transform CL to Cl and so on
+                element = guess_atom_element(atom.name).capitalize()
+            rdatom = Chem.Atom(element)
+            # add properties
+            mi = Chem.AtomPDBResidueInfo()
+            for attr, rdattr in RDATTRIBUTES.items():
+                try: # get value in MDA atom
+                    value = getattr(atom, attr)
+                except AttributeError:
+                    pass
+                else:
+                    if isinstance(value, np.generic):
+                        # convert numpy types to python standard types
+                        value = value.item()
+                    if attr == "segid":
+                        # RDKit needs segid to be an int
+                        try:
+                            value = int(value)
+                        except ValueError:
+                            # convert any string to int
+                            # can be mapped back with np.base_repr(x, 36)
+                            value = int(value, 36)
+                    elif attr == "name":
+                        # RDKit needs the name to be properly formated for a
+                        # PDB file (1 letter elements start at col 14)
+                        name = re.findall('(\D+|\d+)', value)
+                        if len(name) == 2:
+                            symbol, number = name
+                        else:
+                            symbol, number = name[0], ""
+                        value = "{:>2}".format(symbol) + "{:<2}".format(number)
+                    # set attribute value in RDKit MonomerInfo
+                    getattr(mi, "Set%s" % rdattr)(value)
+            rdatom.SetMonomerInfo(mi)
+            # TODO other properties (charges)
+            index = mol.AddAtom(rdatom)
+            # map index in universe to index in mol
+            atom_mapper[atom.ix] = index
+
+        try:
+            bonds = ag.bonds
+        except NoDataError:
+            ag.guess_bonds()
```

Comment:
To be discussed, bond guessing isn't one of those I think are really inaccurate, so I'm not as opposed to guessing by default, but do we think it might be a good idea to throw a warning here?